Core/clim-core/commands/parsers.lisp

```diff
@@ -234,9 +237,11 @@
         for (arg ptype-form) in required-args
         collect `(,arg (progn
                          (write-char ,seperator ,stream)
-                         (present (car ,command-args) ,ptype-form
-                                  :stream ,stream)
-                         (pop ,command-args)))
+                         (let ((value (pop ,command-args)))
+                           (if (unsupplied-argument-p value)
+                               (with-text-face (,stream :italic)
+                                 (write-string "<unsupplied>" ,stream))
+                               (present value ,ptype-form :stream ,stream)))))
```

Comment:
Since this appears in generated code and is repeated in this file, I think defining a function `(defun present-argument-value (value ptype stream))` would make sense.

Reply:
fixed and squashed.